models/product-type/src/attribute-definition/attribute-definition-draft/presets/empty.spec.ts

```diff
@@ -13,11 +13,7 @@ it(`should set attributeConstraint, inputHint, and isSearchable to undefined`, (
       en: expect.any(String),
       fr: expect.any(String),
     }),
-    inputTip: expect.objectContaining({
-      de: expect.any(String),
-      en: expect.any(String),
-      fr: expect.any(String),
-    }),
+    inputTip: undefined,
```

Comment:
`inputTip` is [optional](https://docs.commercetools.com/api/projects/productTypes#attributedefinition), so I included it as a possible undefined. @tylermorrisford or @stephsprinkle, do you see any harm in doing so?

Reply:
Seems totally reasonable to me!